pvlib/pvsystem.py

```diff
@@ -1177,6 +1176,15 @@ def _parse_raw_sam_df(csvdata):
 
     df.index = parsedindex
     df = df.transpose()
+    if 'ADRCoefficients' in df.index:
+        ad_ce = 'ADRCoefficients'
+        df.loc[ad_ce] = df.loc[ad_ce].map(lambda x:
+                                          np.array([float(s) for s in
+                                                    [','.join(x[1:-1]
+                                                              .split())]
+                                                    [0].split(',')]))
```

Comment:
Maybe a code comment should go here to explain what this line is supposed to do.  Does is really all have to be done in one line?